nautobot/extras/signals.py

```diff
@@ -471,15 +471,12 @@ def refresh_job_models(sender, *, apps, **kwargs):
         logger.info("Skipping refresh_job_models() as it appears Job model has not yet been migrated to latest.")
         return
 
-    import_jobs_as_celery_tasks(app)
+    import_jobs()
 
     job_models = []
-    for task in app.tasks.values():
-        # Skip Celery tasks that aren't Jobs
-        if not isinstance(task, JobClass):
-            continue
 
-        job_model, _ = refresh_job_model_from_job_class(Job, task.__class__)
+    for job_class in get_jobs().values():
```

Comment:
You could remove the above `import_jobs()` if you just pass `reload=True` to this?

```suggestion
    for job_class in get_jobs(reload=True).values():
```

Reply:
True. I was trying to keep the diffs ever-so-slightly easier to review. :wink: